stretch_gazebo/nodes/keyboard_teleop_gazebo

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import math
+import keyboard as kb
+
+import rospy
+from sensor_msgs.msg import JointState
+from control_msgs.msg import FollowJointTrajectoryGoal, FollowJointTrajectoryAction
+from trajectory_msgs.msg import JointTrajectoryPoint
+from geometry_msgs.msg import Twist
+import actionlib
+
+
+# NOTE: FUNMAP services and capabilities won't work with Stretch Gazebo simulation
+
+class GetKeyboardCommands:
+    def __init__(self):
+        self.step_size = 'medium'
+        self.rad_per_deg = math.pi / 180.0
+        self.small_deg = 3.0
+        self.small_rad = self.rad_per_deg * self.small_deg
+        self.small_translate = 0.005  # 0.02
+        self.medium_deg = 6.0
+        self.medium_rad = self.rad_per_deg * self.medium_deg
+        self.medium_translate = 0.04
+        self.big_deg = 12.0
+        self.big_rad = self.rad_per_deg * self.big_deg
+        self.big_translate = 0.06
+        self.mode = 'position'  # 'manipulation' #'navigation'
+
+    def get_deltas(self):
+        if self.step_size == 'small':
+            deltas = {'rad': self.small_rad, 'translate': self.small_translate}
+        if self.step_size == 'medium':
+            deltas = {'rad': self.medium_rad, 'translate': self.medium_translate}
+        if self.step_size == 'big':
+            deltas = {'rad': self.big_rad, 'translate': self.big_translate}
+        return deltas
+
+    def print_commands(self, joint_state, command):
+        if command is None:
+            return
+
+        joints = joint_state.name
+
+        def in_joints(i):
+            return len(list(set(i) & set(joints))) > 0
+
+        print('---------- KEYBOARD TELEOP MENU -----------')
+        print('                                           ')
+        if in_joints(['joint_head_tilt']):
+            print('              i HEAD UP                    ')
+        if in_joints(['joint_head_pan']):
+            print(' j HEAD LEFT            l HEAD RIGHT       ')
+        if in_joints(['joint_head_tilt']):
+            print('              , HEAD DOWN                  ')
+        print('                                           ')
+        print('                                           ')
+        print(' 7 BASE ROTATE LEFT     9 BASE ROTATE RIGHT')
+        print(' home                   page-up            ')
```

Comment:
Since Gazebo doesn't have a position mode controller, let's remove this from the menu.

Reply:
The Stretch Base movements from the menu still work. This is achieved by imitating the base movement by [publishing Twist messages](https://github.com/hello-robot/stretch_ros/blob/feature/keyboard_teleop_gazebo_noetic/stretch_gazebo/nodes/keyboard_teleop_gazebo#L191:L192) to the differential drive controller.

Reply:
Got it, thanks!